### DISCUSSION TITLE
need collect metrics from Azure web app for containers

### DISCUSSION THREAD

**Discussion by ashuw018** [tomkerkhove/promitor#1536] [Q&A] [answered]
My Apologies if it sounds like dumb query :)

Is it possible to collect metrics from "Azure WebApp for containers" and then push it to Prometheus ?

**Comment by tomkerkhove** [comment, marked as answer]
No problem at all!

As far as I know, this scraper should do the trick: https://promitor.io/configuration/v2.x/metrics/web-app

**Reply by tomkerkhove** [reply]
Did you manage to get it working @ashuw018?
